pandas/io/feather_format.py

```diff
@@ -106,7 +116,4 @@ def read_feather(path, nthreads=1):
     feather = _try_import()
     path = _stringify_path(path)
 
-    if LooseVersion(feather.__version__) < LooseVersion('0.4.0'):
```

Comment:
why is this removed?

Reply:
I wasn't sure if we would still need it if we would pin a higher version of pyarrow.